pkg/activator/net/throttler.go

```diff
@@ -163,12 +163,18 @@ func newRevisionThrottler(revID types.NamespacedName,
 		revBreaker breaker
 		lbp        lbPolicy
 	)
-	if containerConcurrency == 0 {
+	switch {
+	case containerConcurrency == 0:
 		revBreaker = newInfiniteBreaker(logger)
 		lbp = randomChoice2Policy
-	} else {
+	case containerConcurrency <= 3:
```

Comment:
Do we have benchmarks that warrant this switch? I.e. have we tried round robin for the lower CC values?

Reply:
well the benchmark above shows that full first is better even on code level. Given that pods might be shared (if we don't divide evenly) in case of lower cc we prefer to use the pods at the tail less, since they might be shared, causing queueing.